### PR TITLE
Add GWC API docs for REST improvements

### PR DESCRIPTION
**Pull request by tbarsballe**
API docs for https://github.com/GeoWebCache/geowebcache/pull/593

~WIP pending docs for the new global endpoint (will be added to this PR shortly).~

### CODE REVIEW

**Comment by tbarsballe** [comment]
Added docs for the global endpoint

File: doc/en/api/1.0.0/gwcblobstores.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          description: The name of the blobstore to add or update.
+          type: string
+        - name: blobstoreBody
+          in: body
+          description: The new blobstore definition.
+          required: true
+          schema:
+            $ref: "#/definitions/BlobStore"
+      consumes:
+        - application/xml
```
Comment (johngschulz) [inline doc/en/api/1.0.0/gwcblobstores.yaml]: In swagger editor it doesn’t dropdown the parameter content type application/xml in example, although declared in swagger doc. Might be a swagger limitation.

Comment (tbarsballe) [reply doc/en/api/1.0.0/gwcblobstores.yaml]: Hm, odd. I can reproduce that issue in the swagger editor as well.

However, it seems to work fine on the hosted geoserver docs: http://docs.geoserver.org/latest/en/api/#/1.0.0/gwclayers.yaml

So I think it is just a problem with the editor.

File: doc/en/api/1.0.0/gwcgridsets.yaml [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+            properties:
+              double:
+                type: number
+                description: The resolution value.
+      metersPerUnit: 
+        type: number
+        description: The number of meters per coordinate unit.
+      pixelSixe:
+        type: number
+        description: The assumed pixel size of the dispay device, in meters.
```
Comment (johngschulz) [inline doc/en/api/1.0.0/gwcgridsets.yaml]: Typos "pixelSixe:" (pixelSize)  and " 'dispay' device" (display)

**Comment by tbarsballe** [comment]
Thanks for the review. Typos fixed. Will merge after CI build completes.

**Comment by tbarsballe** [comment]
Backported to 2.13.x

**Pull request merged**